auto_tests/file_saving_test.c

```diff
@@ -75,7 +77,8 @@ static void load_data_decrypted(void)
 
     uint8_t *cipher = (uint8_t *)malloc(size);
     ck_assert(cipher != nullptr);
-    uint8_t *clear = (uint8_t *)malloc(size - TOX_PASS_ENCRYPTION_EXTRA_LENGTH);
+    const size_t clear_size = size - TOX_PASS_ENCRYPTION_EXTRA_LENGTH;
```

Comment:
this might underflow

Reply:
the check above is `0 <= size`, i guess we could change that.

Reply:
Changed the check.